tests/br_key_locked/workload

```diff
@@ -1,4 +1,4 @@
-recordcount=10000
+recordcount=1000
```

Comment:
Is this change intended?

Reply:
It speeds up br_key_locked test, resolving lock is very slow.